glusterd2/volume/struct.go

```diff
@@ -117,6 +120,7 @@ func (v *Volinfo) StringMap() map[string]string {
 	m["volume.transport"] = v.Transport
 	m["volume.auth.username"] = v.Auth.Username
 	m["volume.auth.password"] = v.Auth.Password
+	m["volume.snapList"] = strings.Join(v.SnapList, ",")
```

Comment:
This is only required if it is going to be used in any volfile `{{ volume.snapList }}`